### DISCUSSION TITLE
Facing issues when the library is trying to point to the parent library

### DISCUSSION THREAD

**Discussion by neerajakshid** [Splitties/refreshVersions#535] [Q&A]
I am trying to map org.jetbrains.kotlin:kotlin-android-extensions:_ and I notice the library is not able to detect versions from the versions.properties file.

When assigning the dependency directly I tend to use org.jetbrains.kotlin:kotlin-android-extensions:(version of the parent library org_jetbrains_kotlin). Is there something I am missing? When I run the app, the dependency is not attached to org.jetbrains.kotlin:kotlin-android-extensions: and hence causing issues? Any thoughts?

**Comment by LouisCAD** [comment]
Hello, the version key should be version.kotlin

You can try adding it yourself. Auto-adding Gradle plugin versions is not supported.
